cmd/config/config.go

```diff
@@ -26,6 +26,10 @@ type UpfConfig struct {
 	HeartbeatInterval uint32   `mapstructure:"heartbeat_interval" json:"heartbeat_interval"`
 	HeartbeatTimeout  uint32   `mapstructure:"heartbeat_timeout" json:"heartbeat_timeout"`
 	LoggingLevel      string   `mapstructure:"logging_level" validate:"required" json:"logging_level"`
+	IPPool            string   `mapstructure:"ip_pool" json:"ip_pool"`
+	FTEIDPool         uint32   `mapstructure:"teid_pool" json:"teid_pool"`
+	UEIP              bool     `mapstructure:"ueip" json:"uei"`
```

Comment:
Naming with "feature_" prefix!

Reply:
![image](https://github.com/edgecomllc/eupf/assets/94864219/299fdd23-5a11-4bcd-99e7-a3dfb94d1f60)
I did this yesterday in the very first commit